javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,72 @@
+//Task1: Future age calculator 
+
+let yearOfBirth = 1988;
+// used let to check with other yearofbirth and other futureyear values. 
+let yearFuture = 2030;
+
+const age = yearFuture-yearOfBirth;
+/*
+console.log('\"You will be '+ age + ' years old in ' + yearFuture+ '.\"')
+*/
+const stringText = '\"You will be '+ age + ' years old in ' + yearFuture+ '.\"';
+console.log(stringText)
+
+//TASK2: A dog age calculator
+
+const dogYearOfBirth = 2015; //fixed. No assignment of new values.
+let dogYearFuture = 2025; //can assign and check with other years.
+let dogYear = dogYearFuture-dogYearOfBirth;
+
+let shouldShowResultInDogYears;
+if(shouldShowResultInDogYears == true){
```

Comment:
`shouldShowResultInDogYears` is expected to have the type `boolean`. Comparing it to `true` or `false` can be replaced with the variable itself as the condition.